libs/platform/src/lib/components/form/form-group/form-group.component.html

```diff
@@ -12,7 +12,7 @@
     <div *ngIf="title" class="fd-form-header">
         <span class="fd-form-header__text">{{ title }}</span>
     </div>
-    <div class="fd-container fd-form-layout-grid-container">
+    <div class="fd-container" [class.fd-form-layout-grid-container]="!cozy">
```

Comment:
4. Handle `fd-form-layout-grid-container` class by `compact` property
```suggestion
    <div class="fd-container" [class.fd-form-layout-grid-container]="!compact">
```